src/main/java/de/qabel/desktop/inject/RuntimeDesktopServiceFactory.java

```diff
@@ -104,7 +108,7 @@ public synchronized DropConnector getDropConnector() {
 
     @Override
     public synchronized CrashReportHandler getCrashReportHandler() {
-        return new HockeyApp();
+        return new HockeyApp(getCurrentVersion(), HttpClients.createMinimal());
```

Comment:
with all this version caching etc., you shouldn't return multiple instances but create one and keep it (like the dropDaemon etc.)